lib/oneboxer.rb

```diff
@@ -80,6 +84,33 @@ def self.invalidate(url)
     Discourse.cache.delete(onebox_failed_cache_key(url))
   end
 
+  def self.cache_response_body?(uri)
+    uri = URI.parse(uri) if uri.is_a?(String)
+
+    if SiteSetting.cache_onebox_response_body?
+      SiteSetting.cache_onebox_response_body_domains.split("|").map { |domain| uri.hostname.ends_with?(domain) }.any?
```

Comment:
I _think_ you can pass a block to `any?`

```suggestion
      SiteSetting.cache_onebox_response_body_domains.split("|").any? { |domain| uri.hostname.ends_with?(domain) }
```